logstash.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: logstash
-  version: 8.7.1
```

Comment:
Can you add a comment that 8.7.1-r0 has been withdrawn, so when bumping to 8.7.1 again (if we do that) we should make it r1?

We don't want to republish a withdrawn package, or the next time the withdrawal workflow runs it'll re-withdraw it.

Reply:
yeah good call